centchain/api_test.go

```diff
@@ -1107,7 +1107,7 @@ func TestApi_checkExtrinsicEventSuccess_ExtrinsicSuccess_WithProxySuccess(t *tes
 					Value: registry.DecodedFields{
 						{
 							Value:       nil,
-							LookupIndex: ProxyExecutedExpectedLookupIndex,
+							LookupIndex: 41,
```

Comment:
Where does this 41 come from?

If it is related to the previous `ProxyExecutedExpectedLookupIndex`? like +1?